cloud_images/centos7/create_base_ami.sh

```diff
@@ -10,159 +10,39 @@ set -o errexit -o nounset -o pipefail
 
 : ${DEVICE:?"ERROR: DEVICE must be set"}
 
-ROOTFS=/rootfs
-PARTITION=${DEVICE}1
+OS_VERSION=7-4.1708.el7.centos
+OS_VERSION_SHORT=7.4.1708
```

Comment:
Previously this was configurable: `export CENTOS_VERSION=${CENTOS_VERSION:-"7.4.1708"}` - should it still be here?

Reply:
The version variables is only used by `create_base_ami.sh` to install the `centos-release` package into the rootfs.  It was previously exported and used to `yum update` the server; `yum update` should not be necessary, as the images should be built from the latest (`7.4` in this case) AMI from the AWS Marketplace (step #1).